### DISCUSSION TITLE
3-line supporting text in `md-list-item`.

### DISCUSSION THREAD

**Discussion by knttnk** [material-components/material-web#4836] [Q&A] [answered]
Hi. Thank you for developing this great project!

How can I put 3 or more lines of supporting text in `md-list-item`? Only 2 lines are displayed even if `multi-line-supporting-text` is set.

Here is my html
```html
<md-list-item 
headline="address" 
supporting-text="long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long address"
multi-line-supporting-text> </md-list-item>
```

**Comment by asyncliz** [comment]
Can you provide a reproduction? It appears correct here: https://lit.dev/playground/#gist=2a8ff2f2eb50f02df01ba0bd96b43e52

<img width="491" alt="Screenshot 2023-09-05 at 9 44 01 AM" src="https://github.com/material-components/material-web/assets/34382620/9529ae17-540c-4e80-a9df-362619144766">

**Comment by knttnk** [comment]
Oh, by "2 lines" I mean "headline and 2 more lines." So I want to display headline and 3 more lines.

In your resulting image the sentence is abbreviated with `… `, but I want to show even the last word `address`.

**Reply by asyncliz** [reply]
I don't think there's a way to show overflowing content without changing the width of the item.

Can you provide a screenshot with what you're expecting?

**Reply by knttnk** [reply]
![image](https://github.com/material-components/material-web/assets/61683744/a2985e97-b4e4-405f-8aaa-44e23e152ba1)

I want to change the number of lines and height, not the width, to display the full text, as in this image.

**Reply by asyncliz** [reply, marked as answer]
Gotcha! That's outside of design's current spec for a list item unfortunately.

I'd recommend extending `ListItem` and overriding `render()` methods to customize it. 

From a design/CSS point, list items themselves are very simple. They're just two text styles and padding based on the content. We should encourage and help with building your own items when they don't fit the strict 1/2/3 line spec'd items.

**Reply by knttnk** [reply]
OK, thank you!
